web/book/src/tutorial/filtering.md

```diff
@@ -1,74 +1,73 @@
-> the number of columns. We call this separation of effects "transform
-> orthogonality". It's goal is to keep transform functions composable by
-> minimizing interference of their effects.
->
-> In SQL, you can see this lack of invariant when an aggregation function is
-> used in the `SELECT` clause. Before, the number of rows was kept constant, but
-> introduction of an aggregation function caused the whole statement to produce
-> only one row (per group).
-
-Filter pick rows based on their values:
+**filter:** The `filter` transform picks (passes through) rows based on their values:
 
 ```
-from albums
-filter artist_id == 2
+from invoices
+filter billing_city == "Berlin"
 ```
 
-Here we are looking for rows corresponding to albums which were produced by
-artist with id 2.
+The resulting table contains all the rows that came from Berlin.
 
-In SQL, `filter` would be expressed with either `WHERE` or `HAVING`, depending
-on where in the pipeline it is used.
+PRQL converts the single `filter` transform to use the SQL `WHERE` or `HAVING` command,
+depending on where it appears in the pipeline it is used.
 
----
+**take:** The `take` transform passes through rows based on their position within the relation (table).
+The position (the set of rows) can be specified in two ways:
 
-Take picks rows based on their position within the relation. The position can be
-specified in two ways:
-
-- a plain number `x`, which will pick first `x` rows, or
+- a plain number `x`, which will pick the first `x` rows, or
 - an inclusive range of rows `start..end`.
 
 ```
-from albums
+from invoices
 take 4
 ```
 
 ```
-from albums
+from invoices
 take 4..7
 ```
 
-Of course, it is possible combine all of what we learned into a single pipeline:
+Of course, it is possible combine all these transforms into a single pipeline:
 
 ```
-from albums
-
-# add a new column
-derive my_column = artist_id + 1
+from invoices
 
-# retain only rows that have my_column equal to 91
-filter my_column == 91
+# retain only rows for orders from Berlin
+filter billing_city == "Berlin"
 
-# skip first 10 rows and take only the next 10
+# skip first 10 rows and take the next 10
 take 11..20
 
-# take only first 5 rows
-take 5
+# take only first 3 rows of *that* result
+take 3
 ```
 
-We did something strange at the end here: first we've taken rows `11..20` and
-then taken first 5 rows within that range. This could be done in a single
-transform `take 11..15`. When compiled to SQL this two ways of expressing the
-same thing actually produce an identical query. This is a nice example of how
-PRQL allows fast data exploration, optimizing your queries so you have the
+We did something a bit odd at the end: first we took rows `11..20` and
+then took the first 3 rows from that result.
+Although this could have been done in a single
+transform `take 11..13`, this is a nice example of how
+PRQL allows fast data exploration.
+
+You have the
```

Comment:
I think we should try and keep a consistent voice, and one way I've found to do that is to never use "You", only use a descriptive voice...

Reply:
Busted! :-) Thanks.